prisma/migrations/20230711072830_added_logged_api_requests/migration.sql

```diff
@@ -0,0 +1,14 @@
+-- CreateTable
+CREATE TABLE "LoggedApiRequests" (
+    "id" SERIAL NOT NULL,
+    "createdAt" TIMESTAMP(3) NOT NULL DEFAULT CURRENT_TIMESTAMP,
+    "updatedAt" TIMESTAMP(3) NOT NULL,
+    "ip" TEXT NOT NULL,
+    "appId" INTEGER NOT NULL,
+    "publicKey" TEXT NOT NULL,
```

Comment:
```suggestion
```

Please merge the migrations into one file.

Reply:
done